package Exception;

public class RuntimeExceptin {


    public static void main(String[] args) {
        int arr[] =new int[7];
        System.out.println(arr[8]);
    }
}
